frontend/src/pages/agreements/CreateAgreement.jsx

```diff
@@ -42,6 +43,7 @@ export const CreateAgreement = () => {
                 <StepSelectProject wizardSteps={wizardSteps} />
                 <StepCreateAgreement wizardSteps={wizardSteps} />
                 <StepCreateBudgetLines wizardSteps={wizardSteps} />
+                <StepReviewAgreement />
```

Comment:
I'm not sure if we want to add it as part of CreateAgreement or for it to be a stand alone page 🤔

Reply:
Lets roll with it for now, and then update once the AgreementList is added, we'll move it there.